ttmar_starter.html

```diff
@@ -61,5 +61,13 @@
 
 </body>
 <script>
-
+var firstEvent = document.getElementById("first_paragraph").addEventListener("click", function(){
+    document.body.style.background = "#ddd"
+})
```

Comment:
Not a bad start, and it looks like it should work to me!

I'd note that it's uncommon to store the `event` in a variable. More common might be to store the element in a variable and use that var to add the event:

```js
var firstElement = document.getElementById("first_paragraph");

firstElement.addEventListener("click", function(){
    document.body.style.background = "#ddd"
});
```